pkgs/applications/gis/qgis/unwrapped.nix

```diff
@@ -96,27 +101,40 @@ in mkDerivation rec {
     qscintilla
     qtserialport
     qtxmlpatterns
+    qt3d
+    pdal
+    zstd
   ] ++ lib.optional withGrass grass
```

Comment:
`grass` is currently broken on darwin, due to #126101. Suggest `withGrass` -> `(!stdenv.isDarwin && withGrass)` in all cases for now.

Reply:
Upon further thought, it's not totally clear to me why grass should fundamentally be dependent on webkitgtk. I suppose this is still a stopgap to get qgis to even build on darwin, while it might be possible to go deeper into grass/wxwidgets/etc in parallel to figure out how darwin can still keep grass even while leaving out webkitgtk while it's broken.

Reply:
It turns out it's actually a pretty small fix: wxwidgets just shouldn't ever build webkitgtk while webkitgtk is broken (https://github.com/NixOS/nixpkgs/blob/a0dbe47318bbab7559ffbfa7c4872a517833409f/pkgs/development/libraries/wxwidgets/3.1/default.nix#L62). I'll submit a PR for that so that no workaround here is needed.

Reply:
See #150908 for the wxwidgets fix. There's still a few other issues with grass, so if those can't get resolved then it still may be better to disable grass to get QGIS to still build at all.

In addition, `pdal` also doesn't work on darwin, due to its dependence on https://github.com/asmaloney/libE57Format, which currently is linux only (see https://github.com/asmaloney/libE57Format/issues/48). I'm not very versed in how static versus dynamic libraries play a role in nix, so I'm not sure what is needed to make that derivation cross-platform compatible. Until then, `pdal` should probably be disabled and excluded as a build input for darwin.